diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index e24182d7e23ad859004536c4a9f4b165f71d1419..8c6c87804488c21f0dc6acda3861683c5ed5efb5 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7871,6 +7871,7 @@ const FeatureEntry kFeatureEntries[] = {
      FEATURE_VALUE_TYPE(features::kDragAndDrop)},
 #endif  // BUILDFLAG(IS_ANDROID)
 
+     BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
     {"autofill-enable-update-virtual-card-enrollment",
      flag_descriptions::kAutofillEnableUpdateVirtualCardEnrollmentName,
      flag_descriptions::kAutofillEnableUpdateVirtualCardEnrollmentDescription,
